### ISSUE TITLE
comment engine

### ISSUE THREAD

**Issue by mhurwi** [OpenHunting/openhunt#44]
Seems reasonable to explore some off-the-shelf options, to get a solid system in place quick.

Demand is high, see #4

Disqus and Livefyre are the two commercial options that come to mind.

@marccantwell is looking into it.

**Comment by nblackburn** [comment]
+1 for Disqus, Livefyre has changed a lot since inception and am not sure it's free anymore.

One downside to an out of the box system is the loss of some control as it can't be tied to accounts or the possible karma system (#9).

**Comment by nblackburn** [comment]
One downside to a out of the box system is you would need to manage it remotely and it can't be tied to accounts or the possible karma system (#9).

**Comment by jacquescrocker** [comment]
yeah, with our moderator tools, it'd be tricky to integrate that with an external system

maybe just a simple one we build ourselves would work. we can use an api for spam detection, and let moderators flag comments that are spam or inappropriate

**Comment by nblackburn** [comment]
@jacquescrocker Should be easy enough to pull off and there are plenty of spam prevention services knocking around to integrate with.
